### PR TITLE
Test Secureboot state API

### PR DESCRIPTION
**Pull request by benjamreis**
See: https://github.com/xapi-project/xen-api/pull/5566

### CODE REVIEW

**Comment by benjamreis** [comment]
We need to wait for Upstream merge and our release of th patch before merge.

File: tests/uefi_sb/test_sb_state.py
Comment (benjamreis) [reply tests/uefi_sb/test_sb_state.py:73]: Why? The purpose of this test is to make sure a VM is in `setup_mode` when only PK is missing.

Comment (stormi) [reply tests/uefi_sb/test_sb_state.py:72]: It's also in setup mode when all certs are missing, and that's the most common case for setup mode (that's what you get when you switch a VM to setup mode), so I think it's not useless to also check the return value of the API method in this other case.

**Review by stormi — APPROVED** [review]

**Pull request merged**
